### PR TITLE
add exception traceback of remote function calling

### PR DESCRIPTION
**Pull request by zenghsh3**

### CODE REVIEW

**Comment by zenghsh3** [comment]
Before:
image
After:
image

File: parl/remote/tests/remote_test.py [hunk 1]
```diff
@@ -277,6 +283,21 @@ def test_thread_safe_of_remote_module(self):
...
+        server_port = 17781
+        self._setUp(server_port)
+
+        remote_sim = self.remote_manager.get_remote()
+
+        try:
+            remote_sim.will_raise_exeception_func()
+        except RemoteError as e:
+            logger.info('Expected exception: {}'.format(e))
+            # expected
```
Comment (TomorrowIsAnOtherDay) [inline parl/remote/tests/remote_test.py:297]: To assure that traceback information has been captured and sended to the server, you should assert that `Traceback`  is in the error string.

Comment (zenghsh3) [reply parl/remote/tests/remote_test.py:297]: done

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
Looks good to me!

**Pull request merged**